authors/apps/profiles/urls.py

```diff
@@ -0,0 +1,12 @@
+
+from django.urls import path
+
+
+# local imports
+from .views import GetUserProfile, UpdateUserProfile
+
+
+urlpatterns = [
+    path('profiles/<username>', GetUserProfile.as_view(), name='profile'),
```

Comment:
Be specific on what should be entered like/<str:username>

Reply:
noted. let me look into that

Reply:
made the changes